src/client/options.rs

```diff
@@ -1298,6 +1304,7 @@ impl ClientOptions {
             max_pool_size: conn_str.max_pool_size,
             min_pool_size: conn_str.min_pool_size,
             max_idle_time: conn_str.max_idle_time,
+            max_connecting: None,
```

Comment:
We actually do want to support this option in the URI (see [this table](https://github.com/mongodb/specifications/blob/master/source/uri-options/uri-options.rst#list-of-specified-options) of supported options in the URI options spec), so we'll need to add a `max_connecting` field to the `ConnectionString` struct and support parsing/validating it when specified.

Reply:
Solved